### PR TITLE
added Linux version for sysctl

### PR DESCRIPTION
**Pull request by Cvetomird91**
Added a page for Linux specific options of sysctl.

### CODE REVIEW

**Comment by Cvetomird91** [comment]
The Travis-CI build seems to fail before installing the stable NodeJS version and without running the checks.

**Comment by rubenvereecken** [comment]
I reran it.

**Comment by igorshubovych** [comment]
pages/linux/sysctl.md:3: TLDR004 Command descriptions should end in a period

**Comment by Cvetomird91** [comment]
Fixed.

File: pages/linux/sysctl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Get currently open file handlers:
+
+`sysctl fs.file-nr`
+
+- Get limit for simultaneous open files:
+
+`sysctl fs.file-max`
+
+- Apply changes to /etc/sysctl.conf:
```
Comment (notpeter) [inline pages/linux/sysctl.md]: 'from' not 'to'

````
       -p[FILE], --load[=FILE]
              Load   in   sysctl   settings   from   the   file  specified  or
              /etc/sysctl.conf if none given.
````

**Comment by Cvetomird91** [comment]
The last description was edited.

**Comment by ostera** [comment]
Looking good! Thanks @Cvetomird91 :)

**Pull request merged**